uniffi_bindgen/src/lib.rs

```diff
@@ -443,6 +444,20 @@ fn parse_udl(udl_file: &Utf8Path) -> Result<ComponentInterface> {
     ComponentInterface::from_webidl(&udl).context("Failed to parse UDL")
 }
 
+fn format_code_with_rustfmt(path: &Utf8Path) -> Result<()> {
+    let status = Command::new("rustfmt").arg(path).status().map_err(|e| {
+        let ctx = match e.kind() {
+            ErrorKind::NotFound => "formatting was requested, but rustfmt was not found",
+            _ => "unknown error when calling rustfmt",
+        };
+        anyhow!(e).context(ctx)
```

Comment:
Does this context get printed out when `rustfmt` fails?  It seems like you ignore it below, but I could be missing some anyhow magic.

Reply:
It's not ignored, it gets bubbled out of the function with `?` on the next line.